pkg/apis/monitoring/v1alpha1/types.go

```diff
@@ -108,6 +108,10 @@ type MonitoringStackSpec struct {
 	// +kubebuilder:default="120h"
 	Retention monv1.Duration `json:"retention,omitempty"`
 
+	// retentionSize defines the maximum number of bytes used by the Prometheus data.
```

Comment:
Should this perhaps mention how `Retention` and `RetentionSize` interact if both are present?